rector-ci.php

```diff
@@ -28,7 +29,14 @@
                 'Exception',
                 'Doctrine\ORM\EntityManagerInterface',
                 'Doctrine\ORM\EntityManager',
-            ]
+            ],
+        ]]);
+
+    $services->set(PreferThisOrSelfMethodCallRector::class)
+        ->call('configure', [[
+            PreferThisOrSelfMethodCallRector::TYPE_TO_PREFERENCE => [
+                'PHPUnit\Framework\TestCase' => 'self',
```

Comment:
`$this` should be the corretc choice

Reply:
updated use `PreferThisOrSelfMethodCallRector::PREFER_THIS`